### PR TITLE
Auras: Fill in logic for mob-based auras

### PR DESCRIPTION
**Pull request by zach2good**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

Not yet tested, will have time at the weekend.

Side note; iterating a mob's enmity container suuuuuucks.
```cpp
            for (auto [idx, entry] : *PMob->PEnmityContainer->GetEnmityList())
            {
                CBattleEntity* PTarget = entry.PEnmityOwner;
```

### CODE REVIEW

**Comment by ibm2431** [comment]
Wondering if some genericism with var names / loops might result in us being able to combine the logic for the four branches into one, with the if check just determining which list is getting iterated.

**Comment by zach2good** [comment]
I was thinking the same thing, maybe a "generate lists" section, and then "iterate lists" after

**Comment by zach2good** [comment]
Another option is for me to make `EnmityList` iterable, but I'm sure I'll fight `EnmityContainer` and friends another time

File: scripts/globals/geo.lua [hunk 1]
```diff
@@ -8,23 +8,27 @@ require("scripts/globals/status")
...
--- TODO: After elements are aligned in the codebase, this should become:
--- tpz.geo.spawnLuopan = function(player, target, spell, tick_effect, tick_power, target_type)
-tpz.geo.spawnLuopan = function(player, target, modelID, tick_effect, tick_power, target_type, spell)
+tpz.geo.spawnLuopan = function(player, target, spell, tickEffect, tickPower, targetType)
 
     tpz.pet.spawnPet(player, tpz.pet.id.LUOPAN)
     local luopan = player:getPet()
 
+    local element = spell:getElement()
+    local visibleAura = element - 1
```
Comment (ibm2431) [inline scripts/globals/geo.lua:17]: I greatly approve.

**Review by ibm2431 — APPROVED** [review]

**Pull request merged**